src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -9610,6 +9610,124 @@
         }
       }
     },
+    "usage_collector_stats": {
+      "properties": {
+        "total_duration": {
+          "type": "long",
+          "_meta": {
+            "description": "The total execution duration to grab usage stats for all collectors in milliseconds"
+          }
+        },
+        "total_is_ready_duration": {
+          "type": "long",
+          "_meta": {
+            "description": "The total execution duration of the isReady function for all collectors in milliseconds"
+          }
+        },
+        "total_fetch_duration": {
+          "type": "long",
+          "_meta": {
+            "description": "The total execution duration of the fetch function for all ready collectors in milliseconds"
+          }
+        },
+        "is_ready_duration_breakdown": {
+          "properties": {
+            "DYNAMIC_KEY": {
+              "type": "long",
+              "_meta": {
+                "description": "The execution duration of the isReady function for the collector in milliseconds"
+              }
+            }
+          }
+        },
+        "fetch_duration_breakdown": {
+          "properties": {
+            "DYNAMIC_KEY": {
```

Comment:
I think if we report it in an array we'd have to unwrap it to this object structure I am reporting to be able to visualize it and show top values. I can switch it to an array to reduce mapping explosion :+1:

Reply:
I had v3 indexer in mind: it's way easier to create the indexer just once that indexes each duration in separated documents and find the top values that way vs. revisiting the indexer configuration and the visualizations on every release just in case a new collector is added (and there is a new branch in the tree structure).